pytest_factoryboy/fixture.py

```diff
@@ -11,10 +11,7 @@
 
 from inspect import getmodule
 
-if sys.version_info > (3, 0):
-    from inspect import signature
-else:
-    from funcsigs import signature
+from inspect import signature
```

Comment:
you can compress these imports into 1

Reply:
probably worth using isort for this in another PR

Reply:
Still, can you merge this line to the previous import?

Reply:
I went with reorder_python_imports to match pytest

Reply:
Why do we have to match what pytest does?
Python allows you to have comma separated imports on each line, why do we have to keep them on separate lines just because a tool does that?

Reply:
I think it's best to match the style pytest uses

Reply:
I'm sorry, it doesn't seem reasonable to me. If we want to use a tool like reorder_python_imports, let's discuss it in a separate PR. 

But in this one let's keep imports in the usual style `from ... import xxx, yyy, zzz`